docs/Linting.md

```diff
@@ -0,0 +1,66 @@
+# Linting
+
+[Linting](<http://en.wikipedia.org/wiki/Lint_(software)>) is important part of
+code development that provides static analysis and helps to find bugs in code. It
+also helps to developers to adhere to varios style guidelines during the coding
+stage, rather than only finding out at review time.
+
+It is recommended for any new project to have linting set up from the start.
+
+# ESLint - Javascript Linting
+
+This respository is has [ESLint](http://eslint.org) for providing javascript
```

Comment:
"This respository (sic) is has ESLint..." sounds like it's missing words, or has bonus words.
Probably better as "This repository uses ESLint..."